### PR TITLE
[DOCS-4815] Add clarification to log message remapper

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Adds clarification to log message remapper.

### Motivation

[DOCS-4815](https://datadoghq.atlassian.net/browse/DOCS-4815), SE request (support tickets).

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4815]: https://datadoghq.atlassian.net/browse/DOCS-4815?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/log_configuration/processors.md [hunk 1]
```diff
@@ -237,7 +237,11 @@ Use the [Datadog Log Pipeline API endpoint][1] with the following log service re
 
 ## Log message remapper
 
-Message is a key attribute in Datadog. It is displayed in the message column of the Log Explorer to give context to a log. You can use the search to find a log by log message. Use the log message remapper processor to define one or more attributes as the official log message.
+Message is a key attribute in Datadog. It is displayed in the message column of the Log Explorer to give context to a log. You can use the search to find a log by the log message.
```
Comment (maycmlee) [inline content/en/logs/log_configuration/processors.md:240]:
```suggestion
`message` is a key attribute in Datadog. Its value is displayed in the **Content** column of the Log Explorer to provide context on the log. You can use the search to find a log by the log message. 
```

File: content/en/logs/log_configuration/processors.md [hunk 2]
```diff
@@ -237,7 +237,11 @@ Use the [Datadog Log Pipeline API endpoint][1] with the following log service re
 
 ## Log message remapper
 
-Message is a key attribute in Datadog. It is displayed in the message column of the Log Explorer to give context to a log. You can use the search to find a log by log message. Use the log message remapper processor to define one or more attributes as the official log message.
+Message is a key attribute in Datadog. It is displayed in the message column of the Log Explorer to give context to a log. You can use the search to find a log by the log message. 
+
+Use the log message remapper processor to define one or more attributes as the official log message. Define more than one attribute for cases when the attributes might not exist and an alternative is available. For example, if the defined message attributes are `attribute1`, `attribute2`, and `attribute3`, and `attribute1` does not exist, then `attribute2` is used. Similarly, if `attribute2` does not exist, then `attribute3` is used.
```
Comment (maycmlee) [inline content/en/logs/log_configuration/processors.md:242]:
```suggestion
Use the log message remapper processor to define one or more attributes as the official log message. Define more than one attribute for cases where the attributes might not exist and an alternative is available. For example, if the defined message attributes are `attribute1`, `attribute2`, and `attribute3`, and `attribute1` does not exist, then `attribute2` is used. Similarly, if `attribute2` does not exist, then `attribute3` is used.
```

File: content/en/logs/log_configuration/processors.md [hunk 3]
```diff
@@ -237,7 +237,11 @@ Use the [Datadog Log Pipeline API endpoint][1] with the following log service re
 
 ## Log message remapper
 
-Message is a key attribute in Datadog. It is displayed in the message column of the Log Explorer to give context to a log. You can use the search to find a log by log message. Use the log message remapper processor to define one or more attributes as the official log message.
+`message` is a key attribute in Datadog. Its value is displayed in the **Content** column of the Log Explorer to provide context on the log. You can use the search to find a log by the log message.
```
Comment (alai97) [inline content/en/logs/log_configuration/processors.md:240]:
```suggestion
`message` is a key attribute in Datadog. Its value is displayed in the **Content** column of the Log Explorer to provide context on the log. You can use the search bar to find a log by the log message. 
```

**Review by alai97 — APPROVED** [review]
Looks good, one tiny suggestion!

**Pull request merged**